docs/src/main/asciidoc/adapters/gcp-intro.adoc

```diff
@@ -104,6 +115,8 @@ First, add the Shade Plugin configuration to generate a fat jar when you run the
 </plugin>
 ----
 
+IMPORTANT: If both Spring Boot plugin and Shade plugin are present, Shade plugin may be shading a Spring Boot produced JAR, resulting a Fat JAR that's unusable in Google Cloud Function. Don't forget to remove the Spring Boot plugin!
```

Comment:
Honestly, I haven't seen this. Is it also the case for AWS, which also uses Shade?

```suggestion
IMPORTANT: If both Spring Boot plugin and Shade plugin are present, Shade plugin may be shading a Spring Boot produced JAR, resulting in a Fat JAR that's unusable in Google Cloud Function. Don't forget to remove the Spring Boot plugin!
```